spec/unit/owner_spec.rb

```diff
@@ -17,19 +15,10 @@ module Pod::TrunkApp
 
     it 'finds an existing owner by email' do
       lambda do
-        Owner.find_or_create_by_email_and_update_name(@owner.email, '').should == @owner
+        Owner.find_or_initialize_by_email_and_name(@owner.email, '').should == @owner
       end.should.not.change { Owner.count }
     end
 
-    it "updates an owner's name if specified" do
```

Comment:
in that case, don't these need to be moved into the sessions controller where that check is still being made?